core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/domain/Concept.java

```diff
@@ -104,8 +106,19 @@ public void setIconId(String iconId) {
 		this.iconId = iconId;
 	}
 	
-	public void setAlternativeTerms(SortedSet<String> alternativeTerms) {
-		this.alternativeTerms = alternativeTerms;
+	/**
+	 * @param alternativeTerms
+	 * @deprecated deserialization only: if descriptions property is set via the constructor during deserialization then this ignores the incoming value, use {@link #setDescriptions(Collection)} instead
+	 */
+	@JsonProperty
+	public void setAlternativeTerms(Collection<Description> alternativeTerms) {
```

Comment:
Are the descriptions all alternative descriptions or do they include the description of the term?